packages/main/src/DateTimePicker.ts

```diff
@@ -193,7 +194,6 @@ class DateTimePicker extends DatePicker implements IFormInputElement {
 	/**
 	 * Read-only getters
 	 */
-
 	get classes() {
```

Comment:
As the comment on 199 states, we should move this to the tsx template, not add more to it and reuse it.